docs/guides/migration-guide.md

```
@@ -0,0 +1,600 @@
+result = process_file(file_path)
+
+# 結果の確認
+if result.is_success():
+    print(result.get_summary())
+```
+
+## 依存関係
+
+- `src.core.schemas.types`: 共通型定義
+- `pydantic`: バリデーション
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)